python/pyfunc-server/examples/echo_http/echo_http.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import merlin
+from merlin.model import ModelType, PyFuncModel
+
+
+class EchoModel(PyFuncModel):
+    def initialize(self, artifacts):
+        pass
+
+    def infer(self, request):
+        logging.info("request: %s", request)
+        return request
+
+
+if __name__ == "__main__":
+    # Run pyfunc model locally without uploading to Merlin server
+    merlin.run_pyfunc_model(
```

Comment:
CMIIW but I don't see any changes for the content on `run_pyfunc_model` function, is it already exist previously? only testing and code formatting.

Reply:
`run_pyfunc_model` is a new function added in this PR.

Reply:
just see it, you may mark this as resolve (I can't)